kubernetes/replicator-gke-cc/docs/index.rst

```diff
@@ -0,0 +1,510 @@
+
+To verify your GKE cluster status:
+
+.. sourcecode:: bash
+
+    gcloud container clusters list
+
+To verify that your ``kubectl`` command is configured with the proper context to control your GKE cluster, run:
+
+.. sourcecode:: bash
+
+    kubectl config current-context
+
+The output of the previous command should be a name with the combination of your GKE project, the region, and the value of the ``Makefile`` variable ``GKE_BASE_CLUSTER_ID`` and your machine username, for example:
+
+.. sourcecode:: bash
+
+    kubectl config current-context
+    gke_gkeproject_us-central1-a_cp-examples-operator-jdoe
+
+Demo Execution
+++++++++++++++
+
+To run the automated demo (estimated running time, 8 minutes):
+
+.. sourcecode:: bash
+
+    make demo
+
+The demo will deploy |cp| leverging |co-long|.   As the various components are deployed, the demonstration will echo the various commands as executing them so you can observe the process.  For example, the deployment message for |ak| will look similar to::
+
+    +++++++++++++ deploy kafka
+    helm upgrade --install --namespace operator --wait --timeout=500 -f examples/kubernetes/gke-base/cfg/values.yaml --set global.provider.region=us-central1 --set global.provider.kubernetes.deployment.zones={us-central1-a} -f examples/kubernetes/replicator-gke-cc/cfg/values.yaml -f examples/kubernetes/replicator-gke-cc/cfg/my-values.yaml  --set kafka.replicas=3 --set kafka.enabled=true kafka examples/kubernetes/common/cp/operator/20190912-v0.65.1/helm/confluent-operator
+    Release "kafka" does not exist. Installing it now.
+    NAME:   kafka
+    LAST DEPLOYED: Mon Oct 28 11:42:07 2019
+    NAMESPACE: operator
+    STATUS: DEPLOYED
+    ...
+    ✔  ++++++++++ Kafka deployed
+
+    +++++++++++++ Wait for Kafka
+    source examples/kubernetes/common/bin/retry.sh; retry 15 kubectl --context gke_project-name_us-central1-a_cp-examples-operator-jdoe -n operator get sts kafka
+    NAME    READY   AGE
+    kafka   0/3     1s
+    kubectl --context gke_project-name_us-central1-a_cp-examples-operator-jdoe -n operator rollout status statefulset/kafka
+    Waiting for 3 pods to be ready...
+    Waiting for 2 pods to be ready...
+    Waiting for 1 pods to be ready...
+    statefulset rolling update complete 3 pods at revision kafka-775f97f98b...
+    ✔  ++++++++++ Kafka ready
+
+The last output message you should see is::
+
+    ✔ Replicator GKE->CC Demo running
+
+Validate
+~~~~~~~~
+
+Verify Control Center's View of Multiple Clusters
++++++++++++++++++++++++++++++++++++++++++++++++++
+
+.. include:: ../../docs/includes/port-forward-c3.rst
+
+Now open a web-browser to http://localhost:12345, and you should see |c3| with 2 Healthy Clusters.
+
+.. figure:: images/c3-dual-clusters.png
+    :alt: c3
+
+Next click on the ``controlcenter.cluster`` and then ``Consumers``, and then ``replicator``.  This will give you a view of the |crep-full| consumer group as it replicates the ``stock-trades`` topics to |ccloud|.
```

Comment:
Add some explanation to this page.  Most people are viewing this for the first time and have no idea what they should be gleaning from it.